docs/about-the-consoles/diamond.md

```diff
@@ -0,0 +1,140 @@
+ select and control cues, programs and other configurable functions.
+
+\(C\) The **System Prompt and Menu Softkeys** have their own screen.
+
+\(D\) The **Programming and setup controls** configure and program the console,
+within which the **attribute touchscreen** shows and sets fixture attributes and 
+the **macro/executor buttons** allow storage of frequent key
+sequences, screen layouts, groups or programs. To the left of these
+controls are the cue list playback controls.
+
+\(E\) Three **Preview screens** are provided for monitoring
+video feeds or can be used as extra touch displays.
+
+
+
+## Touch screen controls
+
+![Diamond-F-G](/docs/images/Diamond-F.png)
+
+\(F\) The **Workspace windows** on the touch screens contain touch buttons for
+fixtures, palettes, groups, playbacks, macros and more in a user-configurable
+layout. You can also set fixture attributes and show information windows.
+
+\(G\) The **System Prompt and Menu Softkeys** screen is the nerve centre of
+the console and shows you what is going on. This display contains touch
+buttons for the main menu which change depending on what the console
+is doing. Softkey commands are shown in the manual 
+like this: \[Edit Times\]
+
+## Playbacks
+
+![Diamond-H](/docs/images/Diamond-H.png)
+
+\(H\) The **Rotary Faders** at the top can be used to store
+a range of program functions including cues, chases, and fixture groups.
+The rotary faders have a level indicator around them which is shown in
+a user-configurable colour (halo colour).
+The touch screen below the faders shows legends indicating their function.
+
+\(I\) **The Page Select buttons** in the
+centre of the faders let you change the left or right fader banks to a different page.
+
+\(J\) Two more sets of **Rotary Faders** are provided above the Playback Faders.
+
+\(K\) The motorised **Playback faders** are the main place to store and play back cues or chases
+but can also be assigned a range of other functions. Each fader has a level indicator
+which can be set to user-configurable halo colour.
+The touch screen above the faders shows information about
+each playback.
+
+\(L\) The **Page Select buttons** let you change the left or right
+fader banks to a different page
+of playbacks (for both the playback faders and the lower rotary faders). 
+
+\(M\) The **Macro Buttons** store frequently used sequences of keypresses
+which can be replayed with a single button press. Playbacks can also be
+stored here. The display immediately above the buttons shows legends for
+the buttons. There is a smaller bank of 6 buttons between the playback faders with
+their own legend screen.
+
+\(N\) The **Playback controls** are used to control the playback of cue
+lists and include a configurable T-bar for the Scene Master function and motorised A-B faders.
+
+## Programming controls
+
+![Diamond-N](/docs/images/Diamond-O.png)
+
+\(O\) The five  **Control wheels** are used to set control values (attributes) for
```

Comment:
Might be worth mentioning the new wheel layer paging buttons to the right of the wheels which allow you to select a wheel layer from Levels / Fade+Delay / Shape parameters / Visualiser / Connected Playback.

Reply:
I think this would be better explained in a later section as this intro is really for the novice to see what is where, and those concepts are yet to be explained. I will put it in when I update the wheels section.

Reply:
I suppose my thinking was that theyr'e a whole new bank of buttons and they're not covered by any of the red boxes - maybe adding a new bullet point for them?

Reply:
OK how about "The buttons to the right of the wheels switch them to other functions such as fade timing,
shape control or visualiser control."

Reply:
sgtm